WordPressAuthenticator/Signin/LoginViewController.swift

```diff
@@ -32,7 +32,13 @@ open class LoginViewController: NUXViewController, LoginFacadeDelegate {
 
         return delegate
     }
-    
+
+    open override var preferredStatusBarStyle: UIStatusBarStyle {
+        // Set to the old style as the default.
+        // Each VC in the unified flows needs to override this to use the unified style.
```

Comment:
Would you add a comment mentioning why we override this in each VC? I know it's because we have independent flows that will be enabled, but I doubt I'll remember that a week from now :)